ui/app/pages/mobile-sync/index.scss

```diff
@@ -0,0 +1,17 @@
+.mobile-welcome-popup {
+  &.intro-popup__popover {
+    background: url(/images/mobile-launch.png) 0 0 no-repeat #fff;
+    /* Overrides the size being modified via media query */
+    width: 328px !important;
+    background-size: contain;
+  }
+
+  .popover-header {
+    background: none;
+    margin: 120px 0 0 0;
+  }
+  
+  button.fa-times {
+    margin-top: -240px;
```

Comment:
This is gross but I'm trying to work within the bounds of the current Popover template.  I'll look at subclassing tomorrow.